src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -764,39 +764,39 @@ void EnIk_PlayCutscene(EnIk* this, GlobalContext* globalCtx) {
 
 void EnIk_UpdateDamage(EnIk* this, GlobalContext* globalCtx) {
     s32 pad;
-    f32 pad2;
-    s32 sp2C;
 
     if (this->actionFunc == EnIk_Idle) {
         return;
     }
     if (this->colliderTris.base.acFlags & AC_BOUNCED) {
-        f32 sin = Animation_GetLastFrame(&gIronKnuckleBlockAnim) - 2.0f;
-        if (this->skelAnime.curFrame < sin) {
-            this->skelAnime.curFrame = sin;
+        f32 frame = Animation_GetLastFrame(&gIronKnuckleBlockAnim) - 2.0f;
+
+        if (this->skelAnime.curFrame < frame) {
+            this->skelAnime.curFrame = frame;
         }
         this->colliderTris.base.acFlags &= ~AC_BOUNCED;
         this->colliderCylinder.base.acFlags &= ~AC_HIT;
-    } else if ((this->colliderCylinder.base.acFlags & AC_HIT)) {
-        sp2C = 0;
+    } else if (this->colliderCylinder.base.acFlags & AC_HIT) {
+        s32 armorBroke = false;
```

Comment:
Needs a newline afterwards. I'm also not sure about the name of this variable, is the tense right?